soft65c02_tester/documentation.md

```diff
@@ -35,6 +35,25 @@ memory load #0x1234 "filename"
 Load the given file into memory at the given address. If the file overflows the memory, an error is raised and the program stops.
 This is the preferred way to load programs to be tested into the tester environment.
 
+#### memory load_atari
+
+```
+memory load_atari "filename.xex"
+```
+
+Loads the given file into memory, as an Atari binary, honouring segments indicating their loading
+locations.
+
+#### memory load_as (Apple Single Format)
+
+```
+memory load_as "filename.com"
```

Comment:
Sorry to bother about that, can we change load_as which I tend to perceive as `LOAD AS something` into something like `LOAD APPLE stuff`. I saw the files in this format are using the `.as` extension so it makes sense for people used to it but for folks who don't know about this format file (like me), I think they would find the `load_as` confusing.

Reply:
yes, happy to change the syntax. I think this would be best, and have apple detect the type of file it's loading:

```
memory load atari "filename"
memory load apple "filename"
```

cc65 only generates an apple single format, the apple double format (2 files, data/info split) are similar, but could be supported as they have different magic numbers if anyone was bothered, but I've never come across them.

I'll update the PR now...